rca/static_src/sass/components/_fees.scss

```diff
@@ -0,0 +1,76 @@
+.fees {
+    $root: &;
+
+    &__section {
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+
+        @include media-query(large) {
+            grid-template-columns: 1fr 1fr 27.5%; // match grid lines
+        }
+
+        &--single {
+            #{$root}__item {
+                padding: $gutter 0;
+
+                @include media-query(large) {
+                    padding: $gutter 0 ($gutter * 1.5);
+                }
+            }
+
+            #{$root}__item--title {
+                grid-column: 1 / span 1;
+                padding: $gutter 0;
+
+                @include media-query(large) {
+                    grid-column: 1 / span 1;
```

Comment:
think this can go and just fall back to the first `grid-column: 1 / span 1`

Reply:
fixed, thanks